src/components/ImageUpload.tsx

```diff
@@ -22,16 +23,49 @@ interface ImageDropProps {
 }
 
 const ImageDrop: FunctionComponent<ImageDropProps> = ({ onDrop }) => {
-  const { getRootProps, getInputProps, isDragActive } = useDropzone({ onDrop });
+  const accept: Accept = {
+    'image/jpeg': ['*'],
+    'image/png': ['*'],
+    'image/gif': ['*'],
+    'image/webp': ['*'],
```

Comment:
webp doesn't work 😬

Reply:
oops ye I just copied this from the webapp